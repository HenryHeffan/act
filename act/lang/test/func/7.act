function g (pint x) : pint
{
  chp {
     self := x*x
  }
}


defproc blah ()
{
  int x;
  chp {
    x := g(true)
  }
}
