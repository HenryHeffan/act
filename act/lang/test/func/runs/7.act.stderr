ERROR: File `7.act', line 13, col 11
	Typechecking failed on expression!
	Function `g': arg #0 has an incompatible type
